### ISSUE TITLE
Install Issue - msbuild

### ISSUE THREAD

**Issue by topherlandry** [transitive-bullshit/ffmpeg-concat#35]
Having an issue during installation. First I needed to install Python. I'm not sure why this is necessary since it's not listed in the req's. But that did allow me to progress further. Then I hit an `msbuild` issue:

```C:\Users\topher\AppData\Roaming\npm\node_modules\ffmpeg-concat\node_modules\gl>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
Warning: unrecognized setting VCCLCompilerTool/MultiProcessorCompilation
Warning: unrecognized setting VCCLCompilerTool/MultiProcessorCompilation
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "4.0".```

I'm less positive about how to work around this and didn't see it in the documentation.

**Comment by transitive-bullshit** [comment]
You do not need to install Python for this library to work, and `msbuild` should never come into the picture.

This is a Node.js library, so you have to in install Node.js >= 8.

**Comment by topherlandry** [comment]
This is what I figured. I’ve got node 10 installed. I can provide full logs later. I was very confused.

**Comment by topherlandry** [comment]
```C:\Users\Scrabble\Videos>npm install -g ffmpeg-concat
C:\Users\Scrabble\AppData\Roaming\npm\ffmpeg-concat -> C:\Users\Scrabble\AppData\Roaming\npm\node_modules\ffmpeg-concat\index.js

> gl@4.1.1 install C:\Users\Scrabble\AppData\Roaming\npm\node_modules\ffmpeg-concat\node_modules\gl
> prebuild-install || node-gyp rebuild

prebuild-install WARN install No prebuilt binaries found (target=10.11.0 runtime=node arch=x64 platform=win32)

C:\Users\Scrabble\AppData\Roaming\npm\node_modules\ffmpeg-concat\node_modules\gl>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
Warning: unrecognized setting VCCLCompilerTool/MultiProcessorCompilation
Warning: unrecognized setting VCCLCompilerTool/MultiProcessorCompilation
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "4.0".
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:12)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Scrabble\AppData\Roaming\npm\node_modules\ffmpeg-concat\node_modules\gl
gyp ERR! node -v v10.11.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gl@4.1.1 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gl@4.1.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Scrabble\AppData\Roaming\npm-cache\_logs\2018-10-04T09_44_42_031Z-debug.log```

Looks like the problem is with the `gl` package. Trying to install the MS C++ package they list on their dependencies fails because the version I'm using is newer. Looking at their issues page it may be because I'm using node 10. There are no prebuilt binaries for `headless-gl` with node 10.

**Comment by jempc** [comment]
Just to say I got the same thing (seems to require Python).  To fix required:
```
npm install --global --production windows-build-tools
npm install --global node-gyp
setx PYTHON "%USERPROFILE%\.windows-build-tools\python27\python.exe"
npm install -g ffmpeg-concat
```
